mmv1/third_party/terraform/services/securitycenterv2/resource_scc_v2_notification_config_test.go

```diff
@@ -0,0 +1,139 @@
+package securitycenterv2_test
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/resource"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/terraform"
+	"github.com/hashicorp/terraform-provider-google/google/acctest"
+	"github.com/hashicorp/terraform-provider-google/google/envvar"
+	"github.com/hashicorp/terraform-provider-google/google/tpgresource"
+	transport_tpg "github.com/hashicorp/terraform-provider-google/google/transport"
+	"google.golang.org/api/googleapi"
+	"google.golang.org/api/securitycenter/v1"
+)
+
+func TestAccSecurityCenterOrganizationNotificationConfig_basic(t *testing.T) {
+	t.Parallel()
+
+	context := map[string]interface{}{
+		"org_id":        envvar.GetTestOrgFromEnv(t),
+		"topic_name":    acctest.RandString(t, 10),
```

Comment:
I think you can remove `topic_name` and `config_id` because they aren't being used

Reply:
config_id is used in the test case.